### PR TITLE
feat-#146: Added Admin UI Screens to Frontend

### PR DESCRIPTION
**Pull request by devsharmagit**
## Summary

This PR adds the admin UI screens to the frontend

## Description

Admin UI screens are added to the frontend with responsive and mobile optimization.
/admin/users /admin/blogs are the routes

## Images

image
image

## Issue(s) Addressed

Closes: #146 

## Prerequisites

- [x] Have you followed all the [CONTRIBUTING GUIDELINES](https://github.com/krishnaacharyaa/wanderlust/blob/main/.github/CONTRIBUTING.md#guidelines-for-contributions)?

### CODE REVIEW

**Comment by krishnaacharyaa** [comment]
Please run `npm run format` in the frontend directory,
Please make sure it is dark mode proof, if it seems to take time create a new story for the same.
image

Take care of this error in the deployment :)

Thank you for the PR, looks very neat !!

**Comment by devsharmagit** [comment]
I will solve this error and add dark mode styles as well.
I will submit it tonight.
do I just have to commit to this branch and raise another pr?

**Comment by krishnaacharyaa** [comment]
@devsharmagit please fix the build issues as mentioned above so that we can merge this

**Comment by devsharmagit** [comment]
@krishnaacharyaa 
I have committed to the dark mode functionality.
check and let me know if everything is all right or not?

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]
Looks clean to me, if possible kindly take care of the mobile screen, it would be a good challenge and makes you perfect in tailwind.

File: frontend/src/components/admin-sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { NavLink } from 'react-router-dom';
+import UserIcon from '@/assets/svg/user-icon';
+import BlogIcon from '@/assets/svg/blog-icon';
+import BarIcons from '@/assets/svg/bars-icon';
+import { useState } from 'react';
+import CloseIcon from '@/assets/svg/close-icon';
+
+const AdminSidebar = () => {
+  const [isSidebarOpen, setIsSidebarOpen] = useState<boolean>(true);
+
```
Comment (krishnaacharyaa) [inline frontend/src/components/admin-sidebar.tsx:10]: Shouldn't this be false by default?

File: frontend/src/components/admin-container.tsx
Comment (krishnaacharyaa) [inline frontend/src/components/admin-container.tsx:1]: I guess we are not using this component. But lets use this in my opinion.

File: frontend/src/pages/admin-blogs.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import TrasnIcon from '@/assets/svg/trash-icon';
+import AdminSidebar from '@/components/admin-sidebar';
+import { imageUrls } from '@/constants/images';
+
+const AdminBlogs = () => {
+  return (
+    <>
+      <div className="relative flex flex-grow flex-col bg-light dark:bg-dark md:flex-row">
+        <AdminSidebar />
+        <div className="w-full p-3 px-5 md:p-12">
```
Comment (krishnaacharyaa) [inline frontend/src/pages/admin-blogs.tsx:11]: Adding AdminSidebar in all the blogs will be redundant and not a good practice. So we would be needing the wrapper

File: frontend/src/App.tsx [hunk 3]
```diff
@@ -19,6 +21,10 @@ function App() {
             <Route path="details-page/:title/:postId" element={<DetailsPage />} />
             <Route path="signin" element={<SignIn />} />
             <Route path="signup" element={<SignUp />} />
+            <Route path="admin">
+              <Route path="users" element={<AdminUsers />} />
```
Comment (krishnaacharyaa) [inline frontend/src/App.tsx:25]: This is good, similarly do we have a hack to eliminate the wrapper in the `app.tsx`? Would be a good one if we could get it, else we'll use the wrapper AdminContainer

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]
Thank you for the quick response, couple of reviews if you could address them

**Comment by krishnaacharyaa** [comment]
Hi @devsharmagit, by when can we close this?

**Comment by devsharmagit** [comment]
@krishnaacharyaa 
just commited the changes

**Review by krishnaacharyaa — APPROVED** [review]
Thank you @devsharmagit, LGTM. Merging right away :)

**Pull request merged**